src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -237,6 +237,18 @@ private void storeInPluginContext(List<ImageConfiguration> resolvedImages, Strin
         }
     }
 
+    protected void clearResolvedImages() {
+        Map ctx = getPluginContext();
+        ctx.remove(CONTEXT_KEY_RESOLVED_IMAGES);
+    }
+
+    protected List<ImageConfiguration> recreateResolvedImages() {
```

Comment:
its used in fabric8:watch ;)

it will be used in docker:watch if we decide to push up the changes there too to deal with timestamps